pkg/api/trillian_client.go

```diff
@@ -216,7 +217,9 @@ func (t *TrillianClient) getLeafAndProofByIndex(index int64) *Response {
 		})
 
 	if resp != nil && resp.Proof != nil {
-		if err := t.verifier.VerifyInclusionAtIndex(&root, resp.Leaf.LeafValue, index, resp.Proof.Hashes); err != nil {
+		leaf := t.verifier.BuildLeaf(resp.Leaf.LeafValue)
```

Comment:
we already have `resp.Leaf` at this point; why do we rebuild it here?

Reply:
thanks! removed